config.yaml

```diff
@@ -72,6 +72,13 @@ clients:
       blockwiseTransfer:
         enabled: true
         blockSize: "1024"
+      ownershipTransfer:
+        method: justWorks
+        manufacturer:
```

Comment:
I would propose to use full name, as it's in the OCF. `Manufacturer Certificate` -> `manufacturerCertificate`

Reply:
Fixed